pkg/typesystem/typesystem.go

```diff
@@ -181,11 +184,17 @@ func New(model *openfgav1.AuthorizationModel) *TypeSystem {
 		relations[typeName] = tdRelations
 	}
 
+	uncompiledConditions := make(map[string]*condition.EvaluableCondition, len(model.GetConditions()))
```

Comment:
`uncompiledConditions`, `evaluableCondition`... can agree on one name please? i prefer `uncompiled` here because it's clear that there's an extra step that needs to happen before it's usable

Reply:
the idea here is that: `uncompiledCondition` is an EvaluableCondition in its empty state (haven't called .Compiled()`) and `compiledCondition` after.